pip/wheel.py

```diff
@@ -44,6 +44,20 @@
 logger = logging.getLogger(__name__)
 
 
+class Cache(object):
+    """A cache of wheels for future installs."""
+
+    def __init__(self, cache_dir):
+        """Create a wheel cache.
+
+        :param cache_dir: The root of the cache.
+        """
+        self._cache_dir = cache_dir
+
+    def cached_wheel(self, link):
```

Comment:
Are there going to be other functions on this object? If not would it make sense to just make a ``__call__`` method? I'm fine either way, just throwing it out there.